.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,156 @@
+### Contributing to ZFS on Linux
+
+<p align="center"><img src="http://zfsonlinux.org/images/zfs-linux.png"/></p>
+
+*First of all, thank you for taking the time to contribute!*
+
+By using the following guidelines, you can help us make ZFS on Linux even better.
+
+#### Table Of Contents
+
+[What should I know before I get started?](#what-should-i-know-before-i-get-started)
+
+  * [Get ZFS](#get-zfs)
+  * [Debugging ZFS](#debugging-zfs)
+  * [Where can I ask for help?](#where-can-I-ask-for-help)
+  
+[How Can I Contribute?](#how-can-i-contribute)
+
+  * [Reporting Bugs](#reporting-bugs)
+  * [Suggesting Enhancements](#suggesting-enhancements)
+  * [Pull Requests](#pull-requests)
+  * [Testing](#testing)
+  
+[Style Guides](#style-guides)
+ 
+  * [Coding Conventions](#coding-conventions)
+
+Helpful resources
+
+  * [ZFS on Linux wiki](https://github.com/zfsonlinux/zfs/wiki)
+  * [OpenZFS Documentation](http://open-zfs.org/wiki/Developer_resources)
+
+
+## What should I know before I get started?
+
+### Get ZFS
+
+You can build fresh zfs packages using 
+[this instruction](https://github.com/zfsonlinux/zfs/wiki/Building-ZFS), 
+or use stable packages from 
+[your distribution repository](https://github.com/zfsonlinux/zfs/wiki/Getting-Started).
+
+### Debugging ZFS
+
+The `zdb` utility is the primary tool to debug ZFS. 
+
+Additionally, see 
```

Comment:
Should be `Additionally, using the `--enable-debug` flag when building zfs will enable all ASSERTs and additional correctness tests. Refer to the [build instructions](https://github.com/zfsonlinux/zfs/wiki/Building-ZFS#configure-and-build) for more information.`